src/tools/gophish_cleaner.py

```diff
@@ -2,23 +2,25 @@
 
 """GoPhish Cleaner for a Phishing Campaign Assessment (PCA).
 
+A tool to remove aspects of an Assessment or a campaign from GoPhish.
```

Comment:
I think `Assessment` is unnecessarily capitalized here.  I see other instances of this (e.g. `Campaign` instead of `campaign`) throughout this repo- please clean those up as you come across them.